wordpress.org/public_html/wp-content/themes/pub/wporg-support/helphub-update-php-strings.php

```diff
@@ -56,20 +56,27 @@ function wporg_support_filter_update_php_content( $content ) {
 	$content .= '<p>';
 	$content .= sprintf(
 		/* translators: %s: link URL about keeping WordPress up to date */
-		__( 'PHP is the coding language WordPress is built on, and its version is set at the server-level by your hosting company. Whilst you may be familiar with the importance of <a href="%s">keeping WordPress, and your themes and plugins up-to-date</a>, keeping PHP up-to-date is just as important.', 'wporg-forums' ),
+		__( 'PHP is one of the programming languages that WordPress is built on. The PHP version used on your website is usually set by your hosting company, but may be an option you can adjust yourself. And while you may be familiar with the importance of <a href="%s">keeping WordPress, and your themes and plugins up-to-date</a>, keeping PHP up-to-date is just as important.', 'wporg-forums' ),
```

Comment:
> The PHP version used on your website is usually set by

Still a bit confusing here, perhaps. PHP is an application installed on the web server. How can the version of an application be set to something? I mean, cannot say things like "The version of Firefox on my computer was set to 105", right? :) 

Thinking that explaining availability of different PHP versions on the server would be easier if it uses familiar terms. For example: 

- PHP is (an application) installed on the server by the hosting company.
- There may be several versions installed.
- You (the site owner) can choose which of these different PHP versions is used for your website.

Then perhaps further down can mention that if only one PHP version is installed on the server, the site owner may be able to request their site to be moved to another server with newer PHP version. Think that was a practice at some hosting companies, at least in the past, but good to confirm.

Reply:
This has been reworded, removing the "set" terminology, and referring to the possibility that there may be multiple PHP versions available to the user (e6e229803761f8025efb1912e7d2edc044e1c72d).